query_tests/cases/in/several_chunks.sql

```diff
@@ -0,0 +1,14 @@
+-- IOX_SETUP: ManyFieldsSeveralChunks
+
+-- validate we have access to information schema for listing system tables
+-- IOX_COMPARE: sorted
+SELECT * from h2o;
+-- Plan will look like:
+--  . Two chunks (one parquet and one from ingester) neither overlap nor contain duplicate
+--     --> scan in one scan node
```

Comment:
```suggestion
--  . One chunk from ingester, neither overlap nor contain duplicate
```

I didn't quite follow how there were two chunks in this scenario

Reply:
Ah, I see 

> |               |     IOxReadFilterNode: table_name=h2o, chunks=2 predicate=Predicate                   |

There are two chunks there